client/src/App.js

```diff
@@ -22,32 +22,52 @@ function App() {
-
-                        <Route path="/view-book/:book_id" element={<ViewBook />} />
-                        {/* <Route path="/view-book" element={<ViewBook book={{title: "1984", genre: "sci-fi", author: "George Orwell", pages: 172}}/>} />   TODO: similar to the above statistics, this needs to be made dynamic */}
-                        <Route path="/book-list" element={<BookList />} />
-                        <Route path="/review" element={<ReviewsList />} />
-                        <Route path='/Login' element={<Login/>} />
-                        {/* Add more routes for additional pages */}
-                    </Routes>
-                </div>
-            </div>
+        <div style={{display: "flex", height: "100vh"}}>
+            <Sidebar className = "app">
+                <Menu>
+                    <MenuItem
+                        component={<Link to="/" className="link" />}
+                        icon={
+                            <MenuOutlinedIcon
+                                onClick={() => {
+                                    collapseSidebar();
+                                    }}
+                                    />
+                                }
+                    >
+                        <h2>BOOKSHELF</h2>
+                    </MenuItem>
+                    <MenuItem
+                        component={<Link to="/statistics"/>}
+                        icon={<AssessmentOutlinedIcon />}
+                    >
+                        Statistics
+                    </MenuItem>
+                    <MenuItem
+                        component={<Link to="/book-list" />}
+                        icon={<ListIcon />}
+                    >
+                        Book List
+                    </MenuItem>
+                    <MenuItem
+                        component={<Link to="/review" />}
+                        icon={<RateReviewOutlinedIcon />}
+                    >
+                        Review
+                    </MenuItem>
+                </Menu>
+            </Sidebar>
+            <section>
+                <Routes>
+                    <Route exact path="/" element={<Home />} />
+                    <Route path="/statistics" element={<Statistics user_id={1}/>} /> {/* TODO: in sprint 3 - user_id is hardcoded - change user_id to something more dynamic*/}
+                    <Route path="/view-book" element={<ViewBook book={{title: "1984", genre: "sci-fi", author: "George Orwell", pages: 172}}/>} />   {/* TODO: similar to the above statistics, this needs to be made dynamic */}
+                    <Route path="/book-list" element={<BookList />} />
+                    <Route path="/review" element={<ReviewsList />} />
+                    <Route path='/login' element={<Login/>} />
+                </Routes>
+            </section>
+        </div>
```

Comment:
I don't know why Git thinks this is part of my commits, but it isn't. (It does match sprint-3 though)